src/tcgwars/logic/impl/gen4/SecretWonders.groovy

```diff
@@ -1019,7 +1019,7 @@ public enum SecretWonders implements LogicCardInfo {
                       def pkmnCard = pcs.topPokemonCard
                       def tar = pcs.owner.pbg.all.findAll{it != self}.select("Choose a pokemon to attach $self to",pcs.owner)
                       def energyCard
-                      energyCard = specialEnergy(new CustomCardInfo(ELECTRODE_26).setCardTypes(ENERGY, SPECIAL_ENERGY), [[R, D, F, G, W, Y, L, M, P]]) {
+                      energyCard = specialEnergy(new CustomCardInfo(ELECTRODE_26).setCardTypes(ENERGY, SPECIAL_ENERGY), [[R, D, F, G, W, Y, L, M, P, C]]) {
```

Comment:
I'm curious about this one. Most of these energies explicitly state "This card provides C Energy" and have the little colorless icon in the top right. Does Electrode provide C here too because of "every type of energy" or does it not because it does not explicitly add C?

Reply:
This question also applies to older Rainbow energies with the icon from Team Rocket instead of a Colorless Symbol (pre-HG/SS)

Reply:
Clarified on Discord